src/modding/ModManager.cs

```diff
@@ -718,15 +1044,31 @@ private void DisableModPressed()
         }
 
         availableModsContainer.AddItem(selectedMod.InternalName, icon);
+        SetSelectedModToolTip(availableModsContainer);
 
         enabledMods!.Remove(selectedMod);
         notEnabledMods!.Add(selectedMod);
 
         OnModChangedLists();
     }
 
+    private void SetSelectedModToolTip(ItemList selectedContainer)
```

Comment:
This name is completely wrong. This should be called something like `AddTooltipForLastItemFromCurrentMod` which seems more like this should be a method that just calls SetModToolTip with the right values...

Reply:
Though to avoid bugs, I'd rather this method didn't exist and the SetModToolTip was used instead. Also that method should also take in an index as an optional parameter so that it is clear it acts on the last index by default which may not be wanted in some future usage.